schemas/bedrock-authn-token-http.js

```diff
@@ -0,0 +1,170 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const postTokenQuery = {
+  title: 'postTokenQuery',
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    email: {
+      type: 'string'
+    },
+    account: {
+      type: 'string'
+    },
+    hash: {
+      type: 'string'
+    }
+  }
+};
+
+const postToken = {
+  title: 'postToken',
+  type: 'object',
+  additionalProperties: false,
```

Comment:
Each of these schemas needs to have `required` / `anyOf` added as appropriate to ensure that validation fails if the required fields are missing.